app/templates/profile.hbs

```diff
@@ -24,139 +25,124 @@
   <div class="nypr-account-settings-and-member-center">
     <div class="nypr-account-settings-body">
       {{nypr-accounts/basic-card
-        user=model
-        siteName=siteName
-        emailUpdated=(action emailUpdated)
-        authenticate=(action authenticate)
-        requestTempPassword=(action requestTempPassword)
-        emailIsPendingVerification=emailIsPendingVerification}}
+      user=model
+      siteName=siteName
+      emailUpdated=(action emailUpdated)
+      authenticate=(action authenticate)
+      requestTempPassword=(action requestTempPassword)
+      emailIsPendingVerification=emailIsPendingVerification}}
 
       {{#if model.hasPassword}}
-        {{nypr-accounts/password-card
-          helplinkUrl='/forgot'
-          helplinkText='Forgot password?'
-          changePassword=(action changePassword)}}
+      {{nypr-accounts/password-card
+      helplinkUrl='/forgot'
+      helplinkText='Forgot password?'
+      changePassword=(action changePassword)}}
       {{/if}}
 
       {{#if model.facebookId}}
-        {{#nypr-accounts/social-card as |card|}}
-          {{card.connect-button
-            connected=true
-            providerName='Facebook'
-            providerIcon='facebook-official'
-            buttonClass='nypr-social-connect__button--facebook'
-            manageUrl='https://www.facebook.com/settings?tab=applications'
-            }}
-        {{/nypr-accounts/social-card}}
+      {{#nypr-accounts/social-card as |card|}}
+      {{card.connect-button
+      connected=true
+      providerName='Facebook'
+      providerIcon='facebook-official'
+      buttonClass='nypr-social-connect__button--facebook'
+      manageUrl='https://www.facebook.com/settings?tab=applications'
+      }}
+      {{/nypr-accounts/social-card}}
       {{/if}}
 
     </div>
     {{nypr-accounts/membership-card
-      pledges=orders
-      siteName=siteName
-      siteDomain=siteDomain
-      user=model
-      emailIsPendingVerification=emailIsPendingVerification
-      resendVerificationEmail=(if model.hasPassword
-        (action resendVerificationEmail)
-        (action requestTempPassword model.email)
-      )
-      class='nypr-membership-info-body'}}
+    pledgeManagerEnabled=false
```

Comment:
t looks like these changes are from the nypr-accounts library? Is that correct?

Reply:
Yes, this parameter and `customStatusMessage` on the next line were introduced in the `nypr-account-settings` update.